contracts/StrategyAngleUSDC.sol

```diff
@@ -29,11 +31,14 @@ contract StrategyAngleUSDC is BaseStrategy {
 
     bool public isOriginal = true;
 
+    AngleStrategyProxy public strategyProxy;
+
     // variables for determining how much governance token to hold for voting rights
     uint256 public constant MAX_BPS = 10000;
     address public constant weth =
         address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
     uint256 public percentKeep;
+    uint256 public percentLock;
```

Comment:
This is subjective but I prefer us to do the locking manually

Reply:
Yeah, removed